hurumap/templates/profile/profile_detail.html

```diff
@@ -873,10 +873,149 @@ <h1>Livestock</h1>
+            chartDecimalPlaces: chartDecimalPlaces,
+            geographyData: geographyData,
+            comparisonLevels: comparisonLevels
+        }
+        try {
+            Charts[i] = Chart(chartstuff);
+        } catch(e) {
+            console.log("Error making chart " + chartID)
+            console.log(e)
+            console.log(chartstuff);
+        }
+    });
+}
+var maps = new ProfileMaps();
+maps.drawMapsForProfile(profileData.geography);
+makeCharts();
+// listen for resize, redraw charts to new dimensions
+var lazyRedrawCharts = _.debounce(function() {
+    window.browserWidth = document.documentElement.clientWidth;
+    window.browserHeight = document.documentElement.clientHeight;
+    makeCharts();
+}, 50);
+$(window).resize(lazyRedrawCharts);
+// for touch devices, still allow context toggle
+$('.stat-row').on('click', '.stat', function() {
+    $(this).find('.context').toggle();
+});
+var parentLinkContainer = null;
+if (!!parentLinkContainer) {
+    // set up the listener for trigger to reveal hidden groups
+    $('#header-box').on('click', '.link-reveal', function() {
+        $(this).hide().next('.hidden').show();
+        return false;
+    });
+    // hit the /parents API endpoint
+    $.getJSON(parentGeoAPI)
+        .done(function(results) {
+            // filter out 'this' from the parents
+            var parents = _.reject(results['parents'], function(p) {
+                return p.relation === 'this';
+            });
+            // list of unique parent sumlev types, maintaining order
+            var parentRelations = _.uniq(_.pluck(parents, 'sumlevel'));
+            // collect parents into individual sumlev groups
+            var parentGroups = _.groupBy(parents, function(d) {
+                return d.sumlevel;
+            });
+            // for each parent sumlev type ...
+            var parentLinkSets = _.map(parentRelations, function(r) {
+                // ... compile a set of links to individual profile pages
+                var parentLinkSet = _.map(parentGroups[r], function(v, k) {
+                    return '<a href="/profiles/' + v.geoid + '-' + slugify(v.display_name) + '/">' + v.display_name + '</a>';
+                });
+                var numParents = parentLinkSet.length;
+                // if more than one of a sumlev type, group behind reveal link
+                if (numParents > 1) {
+                    return '<a href="#" class="link-reveal">'+numParents+' '+ sumlevMap[r]['plural'] +'</a><span class="hidden">'+parentLinkSet.join(', ')+'</span>';
+                } else {
+                    // just one of this sumlev type, so add it to list
+                    return parentLinkSet;
+                }
+            })
+            // push the whole thing into the header box ... thingy
+            parentLinkContainer.html(parentLinksPrefix + parentLinkSets.join(', '));
+            $('body').trigger('glossaryUpdate', parentLinkContainer);
+        });
+}
+
+</script>
```

Comment:
This looks like a lot of JS though to have in the HTML. Is all of it really needed?

Reply:
Yes, it's a lot but it's needed. It's where Wazimap creates the charts so we need to override with our own options.

But, we could move it out of html into to `app.js` or something for better organization and just include it in the `{% block body_javascript_extra %}`.

Reply:
@kilemensi depends on how Wazimap implements this JS. Are we overriding an entire file or just a block?

Reply:
Just the ```{% block body_javascript_extra %}``` block @DavidLemayian.

Reply:
Cool. Then let's do it how Wazimap does it to avoid conflicts.

Reply:
Yeah, this is how [Wazimap](https://github.com/OpenUpSA/wazimap/blob/c39c44c420ff4eff360618d6dbfb821b95166edb/wazimap/templates/profile/profile_detail.html#L123) does it.